### ISSUE TITLE
Fine-tuning Albert large

### ISSUE THREAD

**Issue by SohaKhazaeli** [google-research/google-research#123]
I fine-tuned Albert base on my task but didn't get desired accuracy. Now that I am trying to fine-tune Albert large I get this error:
"Resource exhausted: OOM when allocating tensor with shape[8,512,16,64] and type float on /job:localhost/replica:0/task:0/device:GPU:0 by allocator GPU_0_bfc"

I used single GPU, with 12 GB Memory and also 16 GB (two different attempts).
It is interesting that I can fine-tune Bert base and large on the single gpu with 12GB memory.

**Comment by brightmart** [comment]
if i am not get wrong, albert large require big memory than bert base, it could be like requirement for bert large.

**Comment by SohaKhazaeli** [comment]
Thank you brightmart! I experienced the same thing, but even bert xlarge has a model size smaller than Bert base. It seems computationally it is more expensive but the final model size is smaller.

**Comment by riturajkunwar** [comment]
It is relatively computationally more expensive because it has relatively bigger architecture (number of transformer layers, hidden size etc.), therefore more computation.
It has less parameters because it does parameters sharing and also decoupled the embedding size and hidden size and reduced the embedding size tremendously and consequently reducing the parameters/model size.

**Comment by lanzhzh** [comment]
If you use version 2, the memory consumption should be much smaller and you may be able to finetune the models.

**Comment by SohaKhazaeli** [comment]
I was using version 2, but I haven't succeeded to fine-tune on Albert large.

**Comment by dhruvsakalley** [comment]
Not sure why but it seems these models are tagged as not fine-tunable
![image](https://user-images.githubusercontent.com/3661965/69497286-3afad380-0ea9-11ea-91b4-1f05ad6f0bc4.png)
